src/compiler/types.ts

```diff
@@ -1485,6 +1485,8 @@ module ts {
     export interface TypeInferences {

```

Comment:
We should just add a keyword for this instead of having this comment annotation thing. But sure.